Assets/Scripts/Models/Furniture.cs

```diff
@@ -980,6 +972,33 @@ public string GetJobDescription()
         return string.Empty;
     }
 
+    public IEnumerable<string> GetAdditionalInfo()
+    {
+        yield return string.Format("Hitpoint 18 / 18");
+
+        if (PowerConnection != null)
+        {
+            bool hasPower = HasPower();
```

Comment:
`IsOperating` calculated each time `Update()` is called. And it is same as `HasPower()`
Because `HasPower()` calculates and returns value of `IsOperating`